spec/presenters/result_set_presenter_spec.rb

```diff
@@ -2,20 +2,28 @@
 
 RSpec.describe ResultSetPresenter do
 
-  subject(:presenter) { ResultSetPresenter.new(finder)}
+  subject(:presenter) { ResultSetPresenter.new(finder, view_context)}
 
   let(:finder) do
     OpenStruct.new({
       slug: "/a-finder",
       results: results,
       document_noun: document_noun,
-      total: 2,
+      total: 20,
       facets: [ a_facet, another_facet ],
       keywords: keywords,
       atom_url: "/a-finder.atom",
+      default_documents_per_page: 10,
+      values: {},
+      pagination: OpenStruct.new(
```

Comment:
I realise this is already a common thing in this file, but instead of an `OpenStruct` a lot of the time we can use a `double`.

And either way, it's fine to break these nested ones out into their own variables, too.